README.md

```diff
@@ -56,20 +27,36 @@ To execute, run the following commands in your dbt dev environment:"
       identifier="raw_event_table"
 ) -%}
 
-{{ avo_audit.audit_event_volume(
+{{ avo_audit.test_avo_audit_detect_event_count_anomoly(
     relation=raw_event_relation,
-    end_date="2021-10-25",
-    days_back=10,
-    days_lag=5,
     event_name_column="event",
     event_date_column="ts",
-    event_source_column="client"
+    event_source_column="client",
+    end_date="2021-11-25",
+    n_days=15
 ) }}
 ```
+This will return a view where each row represents a event with a spike or drop in data on any day in the past 15 days.
+
+This can also be set up as an automatic test to be run each day against the raw data to make sure you catch any anomoly as soon as possible.
+
 
+```
+// Example.yml
+
+models: 
+  - name: NAME_OF_RAW_TABLE
+    tests:
+      - avo_audit.avo_audit_detect_event_count_anomoly:
+          event_name_column: <NAME_OF_EVENT_NAME_COLUMN>
+          event_date_column: <NAME_OF_DATE_COLUMN>
+          event_source_column: <NAME_OF_SOURCE_COLUMN> // ios / android / web 
+```
+This test will by default check the past 15 days from yesterday, and it will fail if any rows are returned which indicates an anomoly in event volume data for any event in your raw data.
```

Comment:
```suggestion
This test will by default check the past 15 days from yesterday, and it will fail if any rows are returned which indicates an anomaly in event volume data for any event in your raw data.
```